src/calculateRentalCost.js

```diff
@@ -28,7 +28,8 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  return (days >= 7) ? (days * 40) - 50
+    : (days >= 3) ? (days * 40) - 20 : days * 40;
```

Comment:
[checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 1st item, I don't know what 40, 50, 20, 7 and 3 are